README.md

```diff
@@ -105,6 +105,7 @@ Role Variables
   ansistrano_deploy_to: "/var/www/my-app" # Base path to deploy to.
   ansistrano_version_dir: "releases" # Releases folder name
   ansistrano_current_dir: "current" # Softlink name. You should rarely changed it.
+  ansistrano_current_via: "symlink" # Deployment strategy who code should be deployed to current path. Options are symlink or rsync
```

Comment:
The comment should say something as "Deployment strategy about **how** code should be deployed to current path.